packages/atomic/src/components/result-template-components/atomic-result-value/atomic-result-value.tsx

```diff
@@ -19,15 +28,32 @@ export class AtomicResultValue {
    */
   @Prop() value!: string;
   /**
-   * Which highlight should the value be highlighted with
+   * Which highlight should the value be highlighted with.
+   * Possible values are `firstSentencesHighlights`, `excerptHighlights`, `printableUriHighlights` or `summaryHighlights`.
+   *
    */
   @Prop() shouldHighlightWith?: string;
+  /**
+   * The format in which to display the result value.
+   * Possible values are `text`, `date`, `number` or `currency`.
+   */
+  @Prop() format: ResultValueFormat = 'text';
```

Comment:
I don't think we should repeat the same pattern as the JSUI, because it creates incompatible combinations of options. The `format` needs to be set to the correct string for e.g. `dateFormat` to work. It also means someone who is not displaying dates is still pulling `datejs`.